TreeMaker/python/triggerNameList.py

```diff
@@ -1,49 +1,47 @@
 import FWCore.ParameterSet.Config as cms
 
 triggerNameList = cms.vstring(
-	'HLT_PFMET90_PFMHT90_IDTight_v',
-	'HLT_PFMET100_PFMHT100_IDTight_v',
 	'HLT_PFMET110_PFMHT110_IDTight_v',
 	'HLT_PFMET120_PFMHT120_IDTight_v',
-	'HLT_Ele25_eta2p1_WPTight_Gsf_v ',
+        'HLT_PFMET130_PFMHT130_IDTight_v',
+	'HLT_PFMET140_PFMHT140_IDTight_v',
+        'HLT_Ele28_eta2p1_WPTight_Gsf_HT150_v',
 	'HLT_Ele27_WPTight_Gsf_v',
-	'HLT_Ele27_eta2p1_WPLoose_Gsf_v ',
-	'HLT_Ele45_WPLoose_Gsf_v',
-	'HLT_Ele105_CaloIdVT_GsfTrkIdT_v',
-	'HLT_Ele15_IsoVVVL_PFHT350_PFMET50_v',
+	'HLT_Ele20_eta2p1_WPLoose_Gsf_v ',
+	'HLT_Ele20_WPLoose_Gsf_v',
+	'HLT_Ele115_CaloIdVT_GsfTrkIdT_v',
+	'HLT_Ele15_IsoVVVL_PFHT450_PFMET50_v',
 	'HLT_Ele15_IsoVVVL_PFHT600_v',
-	'HLT_Ele15_IsoVVVL_PFHT350_v',
+	'HLT_Ele15_IsoVVVL_PFHT450_v',
 	'HLT_IsoMu24_v ',
-	'HLT_IsoMu22_eta2p1_v',
+	'HLT_IsoMu24_eta2p1_v',
 	'HLT_Mu50_v',
-	'HLT_Mu15_IsoVVVL_PFHT350_PFMET50_v',
+	'HLT_Mu15_IsoVVVL_PFHT450_PFMET50_v',
 	'HLT_Mu15_IsoVVVL_PFHT600_v',
-	'HLT_Mu15_IsoVVVL_PFHT350_v',
-	'HLT_Photon90_CaloIdL_PFHT500_v',
-        'HLT_Photon90_CaloIdL_PFHT600_v',
+	'HLT_Mu15_IsoVVVL_PFHT450_v',
+	'HLT_Photon90_CaloIdL_PFHT700_v',
         'HLT_Photon135_PFMET100_v',
-	'HLT_Photon165_HE10_v',
+	'HLT_Photon165_R9Id90_HE10_IsoM_v',
 	'HLT_Photon175_v',
-	'HLT_PFHT300_PFMET100_v',
-	'HLT_PFHT300_PFMET110_v',
-	'HLT_PFHT800_v',
-	'HLT_PFHT900_v',
-	'HLT_DoubleMu8_Mass8_PFHT300_v',
-	'HLT_DoubleEle8_CaloIdM_TrackIdM_Mass8_PFHT300_v',
+	'HLT_PFHT500_PFMET100_PFMHT100_IDTight_v',

```

Comment:
We also have MET/MHT only triggers which can be used for the signal trigger for lower MET regions :

        'HLT_PFMETNoMu110_PFMHTNoMu110_IDTight_v',
        'HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_v',
        'HLT_PFMETNoMu130_PFMHTNoMu130_IDTight_v',
        'HLT_PFMETNoMu140_PFMHTNoMu140_IDTight_v',
        'HLT_PFMET110_PFMHT110_IDTight_v',
        'HLT_PFMET120_PFMHT120_IDTight_v',
        'HLT_PFMET130_PFMHT130_IDTight_v',
        'HLT_PFMET140_PFMHT140_IDTight_v',